src/orion/storage/track.py

```diff
@@ -0,0 +1,464 @@
+# -*- coding: utf-8 -*-
+"""
+:mod:`orion.storage.legacy -- Track Storage Protocol
+====================================================
+
+.. module:: base
+   :platform: Unix
+   :synopsis: Implement a storage protocol to allow Orion to use track as a storage method
+
+"""
+
+from collections import defaultdict
+import datetime
+import logging
+import uuid
+
+try:
+    from track.client import TrackClient
+    from track.persistence.utils import parse_uri
+    from track.serialization import to_json
+    from track.structure import CustomStatus, Status as TrackStatus
+    from track.structure import Project, Trial as TrackTrial, TrialGroup
+
+    HAS_TRACK = True
+except ImportError:
+    HAS_TRACK = False
+
+from orion.core.worker.trial import Trial as OrionTrial
+from orion.storage.base import BaseStorageProtocol
+
+
+log = logging.getLogger(__name__)
+
+
+if HAS_TRACK:
+    _status = [
+        CustomStatus('new', TrackStatus.CreatedGroup.value + 1),
+        CustomStatus('reserved', TrackStatus.CreatedGroup.value + 2),
+
+        CustomStatus('suspended', TrackStatus.FinishedGroup.value + 1),
+        # CustomStatus('completed', TrackStatus.FinishedGroup.value + 2),
```

Comment:
Why not completed?

Reply:
Oh, just saw below. There is no official status for any of these in Track beside Completed?

Reply:
already exists in Track, we reused track's completed instead of defining another.

Reply:
And there is no official status for any of these in Track beside Completed?

Reply:
actualyl yes I, updated the status, only 'reserved' and 'new' are custom status

Reply:
Did you push the modifications? I don't see any.

Reply:
not yet